src/entities/friendships.rs

```diff
@@ -1,28 +1,48 @@
 use sqlx::{types::Uuid, Error, Row};
-use std::sync::Arc;
+use std::{fmt, sync::Arc};
 
 use crate::{
     components::database::{DBConnection, DatabaseComponent},
     generate_uuid_v4,
 };
 
+#[cfg_attr(test, faux::create)]
 #[derive(Clone)]
```

Comment:
remove derive

Reply:
1. It's needed because of app usage, it's being cloned for the health components